File: src/main/java/de/maggiwuerze/xdccloader/persistence/UserSettingsRepository.java
package de.maggiwuerze.xdccloader.persistence;

import de.maggiwuerze.xdccloader.model.entity.UserSettings;
import java.util.List;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface UserSettingsRepository extends CrudRepository<UserSettings, Long> {

	List<UserSettings> findAll();
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/persistence/UserSettingsRepository.kt
package de.maggiwuerze.xdccwebloader.persistence

import de.maggiwuerze.xdccwebloader.persistence.entity.UserSettings
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface UserSettingsRepository : CrudRepository<UserSettings, UUID> {
    override fun findAll(): List<UserSettings>
    fun getFirstById(id: UUID): UserSettings?
}